doc/traits/release.rst

```diff
@@ -29,6 +29,19 @@ The following operations are performed:
   Declaring this trait changes the default triggering behaviour to "manual"
 
 
+`release_callback` Attribute
+----------------------------
+
+You can add additional changes in the release commit using a callback script defined by the 
```

Comment:
this should rather go to concourse.model.traits.release.py (into the doc attribute of the corresponding AttributeSpec)

Also, please consider using passive sentences to match the other documentation's style. For examples, kindly refer to e.g. this file's ctx:

"The following operations are performed"
"Declaring this trait changes .."

--

E.g. ```an optional callback that is called during release commit creation. The callback is passed the absolute path to the main repository's work tree via environment variable `REPO_DIR`. Any changes left inside the worktree are added to the resulting release commit.```

Reply:
Done